app/scripts/controllers/landingPage.js

```diff
@@ -84,11 +86,16 @@ angular.module('openshiftConsole')
       }
     }
 
-    function resetRouteParams() {
-      // Reset the route params in the next digest cycle
-      $timeout(function() {
-        $location.replace();
-        $location.search('startTour', null);
-      });
-    }
+    var deregisterLocationStart = $scope.$on('$locationChangeStart', function(event, newUrl) {
+      if (newUrl.indexOf('startTour') > 0) {
+        event.preventDefault();
+        if (tourEnabled) {
```

Comment:
Might be better to only add the location change listener when the tour is enabled.